### PR TITLE
{bio}[gompi/2020a] fpocket v3.1.4.2

### PR DESCRIPTION
**Pull request by bedroge**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
pg-interactive.hpc.rug.nl - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/8e06f124992ea4bf9648b7ae1b5e4810 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node378.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/05215715513c18f2b6dd88796093e225 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node104.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/3fb6ddd668b934774bea3728247903cf for a full test report.

File: easybuild/easyconfigs/f/fpocket/fpocket-3.1.4.2-gompi-2020a.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+source_urls = ['https://github.com/Discngine/fpocket/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['6a801c7a4af0f76a58369b558ca5f6674571e1825024d459bba20bc07ae4f8b7']
+
+dependencies = [('netCDF', '4.7.4')]
+
+# Don't overwrite $LD_LIBRARY_PATH, but append fpocket paths to it.
+prebuildopts = 'sed -i "s/LD_LIBRARY_PATH=/LD_LIBRARY_PATH=\$(LD_LIBRARY_PATH):/g" makefile && '
+
```
Comment (lexming) [inline easybuild/easyconfigs/f/fpocket/fpocket-3.1.4.2-gompi-2020a.eb:21]:
```suggestion
# Use CFLAGS from EB
prebuildopts += 'sed -i "s/-O2.*c99/$CFLAGS/g" makefile && '

```

File: easybuild/easyconfigs/f/fpocket/fpocket-3.1.4.2-gompi-2020a.eb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+name = 'fpocket'
+version = '3.1.4.2'
+
+homepage = 'https://github.com/Discngine/fpocket'
+description = """The fpocket suite of programs is a very fast open source protein pocket detection algorithm based on
+Voronoi tessellation. The platform is suited for the scientific community willing to develop new scoring functions and 
+extract pocket descriptors on a large scale level."""
+
+toolchain = {'name': 'gompi', 'version': '2020a'}
+
```
Comment (lexming) [inline easybuild/easyconfigs/f/fpocket/fpocket-3.1.4.2-gompi-2020a.eb:12]:
```suggestion
toolchainopts = {'usempi': False, 'cstd': 'c99'}

```

**Review by lexming — CHANGES_REQUESTED** [review]
The build of `fpocket` works well but it ignores the `CFLAGS` from EB. I suggest to fix it in the Makefile as well. Please see below

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node381.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/189623f6b69457d870d90ee9000f3496 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node109.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/21023f2506d1ffe590dbacea616c905c for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @bedroge !

**Pull request merged**